src/utils/statusColumnUtils.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  Oppfolgingsgrunn,
+  AktivitetskravStatus,
+} from '@/api/types/personoversiktTypes';
+import { PersonData } from '@/api/types/personregisterTypes';
+
+function mapOppfolgingsgrunn(oppfolgingsgrunn: Oppfolgingsgrunn) {
+  switch (oppfolgingsgrunn) {
+    case Oppfolgingsgrunn.ANNET:
+      return 'Annet';
+    case Oppfolgingsgrunn.FOLG_OPP_ETTER_NESTE_SYKMELDING:
+      return 'Følg opp etter sykmelding';
+    case Oppfolgingsgrunn.FRISKMELDING_TIL_ARBEIDSFORMIDLING:
+      return 'Vurder § 8-5';
+    case Oppfolgingsgrunn.TA_KONTAKT_ARBEIDSGIVER:
+      return 'Kontakt arbeidsgiver';
+    case Oppfolgingsgrunn.TA_KONTAKT_BEHANDLER:
+      return 'Kontakt behandler';
+    case Oppfolgingsgrunn.TA_KONTAKT_SYKEMELDT:
+      return 'Kontakt sykmeldt';
+    case Oppfolgingsgrunn.VURDER_14A:
+      return 'Vurder § 14a';
+    case Oppfolgingsgrunn.VURDER_ANNEN_YTELSE:
+      return 'Vurder annen ytelse';
+    case Oppfolgingsgrunn.VURDER_ARBEIDSUFORHET:
+      return 'Vurder § 8-4';
+    case Oppfolgingsgrunn.VURDER_DIALOGMOTE_SENERE:
+      return 'Vurder dialogmøte';
+    case Oppfolgingsgrunn.VURDER_TILTAK_BEHOV:
+      return 'Vurder tiltak';
+    default:
+      return '';
+  }
+}
+
+function mapAktivitetskravStatus(status: AktivitetskravStatus): string {
+  switch (status) {
+    case AktivitetskravStatus.NY:
+      return '- Ny kandidat';
+    case AktivitetskravStatus.AVVENT:
+      return '- Avventer';
+    case AktivitetskravStatus.FORHANDSVARSEL:
+      return '- Forhåndsvarsel';
+    default:
+      return '';
+  }
+}
+
+function mapArbeidsuforhetStatus(svarfrist: Date | undefined) {
+  const today = new Date();
+
+  if (svarfrist && svarfrist >= today) {
+    return 'Forhåndsvarsel';
+  } else {
+    return 'Vurder avslag';
+  }
+}
+
+export function getStatus(personData: PersonData): string {
+  if (personData.oppfolgingsoppgave) {
+    return `Oppfølgingsoppgave - ${mapOppfolgingsgrunn(
+      personData.oppfolgingsoppgave.oppfolgingsgrunn
+    )}`;
+  }
+  if (personData.harAktivitetskravVurderStansUbehandlet) {
+    return 'Aktivitetskrav - Vurder stans';
```

Comment:
Synes vi også her bør skrive i klartekst hva som har skjedd og ikke foreslått handling 😊
```suggestion
    return 'Aktivitetskrav - Forhåndsvarsel utløpt';
```